bzip2.rs

```diff
@@ -1251,9 +1248,37 @@ unsafe fn compress(name: Option<&str>) {
     delete_output_on_interrupt = false;
 }
 
+enum OutputStream {
+    Stdout(std::io::Stdout),
+    File(std::fs::File),
+}
+
+impl std::io::Write for OutputStream {
+    fn write(&mut self, buf: &[u8]) -> std::io::Result<usize> {
+        match self {
+            OutputStream::Stdout(_stdout) => _stdout.write(buf),
```

Comment:
s/_stdout/stdout here and below

Reply:
stdin and stdout are statics in this commit, and cannot be shadowed. so these can only be renamed after your windows PR

Reply:
actually, we still (on unix) have a static named `stdin`. We could work around that with `link_name`?